qdrant-landing/content/articles/food-discovery-demo.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Food Discovery Demo
+short_description: Feeling hungry? Multimodal semantic search makes it easy to find the perfect meal.
```

Comment:
```suggestion
short_description: Feeling hungry? Find the perfect meal with Qdrant's multimodal semantic search.
```

Reply:
Give me a few more rounds. I really wanna do this well. The demo could be a big hit.